deploy/olm-catalog/performance-addon-operator/4.7.0/performance-addon-operator.v4.7.0.clusterserviceversion.yaml

```diff
@@ -264,3 +264,23 @@ spec:
     targetPort: 4343
     type: ConversionWebhook
     webhookPath: /convert
+  - admissionReviewVersions:
+    - v1beta1
+    containerPort: 443
+    deploymentName: performance-operator
+    failurePolicy: Fail
+    generateName: vwb.performance.openshift.io
+    rules:
+    - apiGroups:
+      - performance.openshift.io
+      apiVersions:
+      - v2
```

Comment:
We only validate v2? We don't validate v1 and v1alpha1? They can be passed as well to PAO

Reply:
@marcel-apf We should block the creation of the new profile with the old version, no?

Reply:
I don't get it, you want to be able to create or modify only v2 profiles ? And fail editing / creation of v1 /v1alpha1 profiles?

Reply:
I mean that once we have 4.8, do we want to allow a user to create profiles with old versions? I believe we want to deprecate the old API version after one or two release versions.

Reply:
But we want them to be able to edit. As long as the user can edit a V1  profile we are fine. The reason is the profile promotion from v1 to v2 is lazy. It is done as soon as the user changes something in the profile.

Reply:
I see, it makes sense, so my plan for 4.8:
1) block the creation of v1 and v1alpha1 
2) provide validation during the update for all versions v2, v1, v1alpha1 WDT?

Reply:
ack